AnnoyancesFilter/sections/popups.txt

```diff
@@ -113,6 +113,7 @@ sodapdf.com##.be-tooltip
 ! Specific sites
 !######### Subscription popups add to subscriptions.txt ######### 
 !
+iobit.com##.popout
```

Comment:
Can you reproduce it? I don't see it on my end.

Reply:
Yes. Here: 
![image](https://user-images.githubusercontent.com/35523759/142605511-0ea66e58-ec14-41e9-bcbd-7dc76bbc38b6.png)

Reply:
Ok. it looks like an ad. What's the end destination? If it leads to another website, please do next.
1. Move your rule to the `Base - specific`.
2. Remove this `iobit.com###itoppopout`.

Reply:
No it doesn't leads to another website, it leads here: https://www.iobit.com/en/driver-booster.php?from=dbsptest